docs/gitrepo-structure.md

```diff
@@ -150,6 +150,21 @@ targetCustomizations:
   clusterGroup: group1
 ```
 
+!!! note 
+    For private helm repo, user can reference a secret with the following key:
+    
+    1. `username` and `password` for basic http auth if helm http repo is behind basic auth.
+    
+    2. `cacerts` for custom CA bundle if helm repo is using custom CA.
+    
+    3. `ssh-privatekey` for ssh private key if repo is using ssh protocol.
+    
+    For example, to add a secret in kubectl, run 
+    
+    `kubectl create secret -n $namespace generic helm --from-literal=username=foo --from-literal=password=bar --from-file=cacerts=/path/to/cacerts --from-file=ssh-privatekey=/path/to/privatekey.pem`
+    
+    After secret is created, specify the secret to `gitRepo.spec.helmSecretName`. Make sure secret is created under the same namespace with gitrepo.
```

Comment:
Is this supposed to be plaintext?
![image](https://user-images.githubusercontent.com/39320683/114252448-3ab28980-9973-11eb-8a5b-563a3c1d0214.png)

Reply:
explained offline that this isn't pure MD. resolving